README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
 - Towers of Doom
 - Warhead Junction
 
-[View Map Pool Here](https://www.chairleague.com/maps/enabled)
-
-Study up, hone your strats, and be ready to throw down on these battlefields on your preferred night of Chair League action.
+[View Map Pool Here](https://www.chairleague.com/maps/enabled).
 
-### Substitutions
-
-Matches require four or five players from your team to check in before each round can start. Teams may sub in one non-rostered player, so long as that player is eligible to play in that league (see [Player on Multiple Teams](#player-on-multiple-teams) and [Selecting a Division](#selecting-a-division)).
+### Hero Status
 
-Substitutions can occur in between rounds. A player who plays in any round is not required to play in any other rounds in the series. 
+Generally, only heroes with significant bugs are banned, at Chair League's discretion. Heroes are available upon release.
 
-Substitutions may not happen during or after draft. Once drafting has begun, the roster is locked. Beware, Abathur mains.
+[View Hero Status here](https://www.chairleague.com/heroes/enabled).
 
-### Player on Multiple Teams
+### Pricing
 
-Players can join and play on multiple teams in different divisions.
+Entry into a league costs $3 per week of regular season play per team, regardless of roster size. Playing the full season costs $24; teams joining late pay only for the weeks remaining in the season. Additionally, if a team has a bye for a week, they will be refunded one week's fee.
```

Comment:
Do we need a pricing section in the rules since it has its own page on the site?